### ISSUE TITLE
Set marker height and width

### ISSUE THREAD

**Issue by jrdn91** [aslamdoctor/jQuery.mapmarker#1]
Is there anyway to get this feature added. I'm trying to use this within a retina environment and want to use larger images and shrink them down with a hard coded value to get my marker icons to look nice on a retina display.

**Comment by aslamdoctor** [comment]
you can simply use any size of images for Marker, it will work straight away.

**Comment by jrdn91** [comment]
yes but what I want to do is use an image that is say 200 x 200 and scale it down to 100 x 100 so that retina devices will see the image crisper
